src/SynchroFeed.Command.ApplicationIs64bit/ApplicationIs64BitCommand.cs

```diff
@@ -94,10 +95,16 @@ public override CommandResult Execute(Package package, PackageEvent packageEvent
                 return new CommandResult(this);
             }
 
-            var result = DoesPackageContain32bitExecutable(Action.SourceRepository, package);
+            if (package.Content == null)
```

Comment:
I noticed that we already do a 'Repository.Fetch' before calling all commands, so I assume we can just assume if the content isn't there.  The package is invalid in some form or another and can be ignored.

Please correct me if I'm wrong.

Reply:
I've adjusted this for the other commands as well.